jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -237,10 +261,6 @@ protected ByteBuffer allocateDirect(int capacity)
         return ByteBuffer.allocateDirect(capacity);
     }
 
-    protected void removed(RetainableByteBuffer retainedBuffer)
-    {
-    }
-
```

Comment:
Are we able to remove this method in a dot release?   
It is probably OK, but kind of bad form.